### DISCUSSION TITLE
does github have a phone number or page where you can report scams?

### DISCUSSION THREAD

**Discussion by zevxis** [community/community#147373] [Code Security]
### Select Topic Area

Question

### Body

i fell victim to crypto thievery tonight by a (purported) user on github, 'Thorazine361'. i couldn't find any specific area to report it. he answered my message for help in recovering one of my wallets referring me to the URL. if anyone could help, i'd appreciate...regards, z

**Comment by Osmia-Atriventris** [comment]
There's a help page regarding reporting users [here](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).

Basically, on their user profile, on the left side of the page under their information, there's a link to block or report the user. Click 'report abuse', and complete the form to tell Github about the behavior.

**Comment by ghostinhershell** [comment]
Hi @zevxis, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
